files/k3s-install-server.sh

```diff
@@ -198,4 +198,17 @@ if [[ "$first_last" == "first" ]]; then
 fi
 %{ endif }
 
-%{ endif }
\ No newline at end of file
+%{ if install_argocd }
+if [[ "$first_last" == "first" ]]; then
+  kubectl create namespace argocd
+  kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/${argocd_release}/manifests/install.yaml
+fi
+
+%{ if install_argocd_image_updater }
```

Comment:
Moved argo-image-updater inside the argocd if, but it is still optional

Reply:
We can save an "if" I think:

```
%{ if install_argocd }
if [[ "$first_last" == "first" ]]; then
  kubectl create namespace argocd
  kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/${argocd_release}/manifests/install.yaml
%{ if install_argocd_image_updater }
  kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj-labs/argocd-image-updater/${argocd_image_updater_release}/manifests/install.yaml
%{ endif }
fi
```

Reply:
Good catch, removed the unneeded